cms/src/main/java/uk/nhs/digital/arc/process/ArcJcrContentNodeBinder.java

```diff
@@ -0,0 +1,101 @@
+package uk.nhs.digital.arc.process;
+
+import org.onehippo.forge.content.pojo.binder.ContentNodeBindingItemFilter;
+import org.onehippo.forge.content.pojo.binder.jcr.DefaultJcrContentNodeBinder;
+import org.onehippo.forge.content.pojo.common.ContentValueConverter;
+import org.onehippo.forge.content.pojo.model.ContentItem;
+import org.onehippo.forge.content.pojo.model.ContentNode;
+import org.onehippo.forge.content.pojo.model.ContentProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.nhs.digital.arc.transformer.abs.AbstractTransformer;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import javax.jcr.Node;
+import javax.jcr.NodeIterator;
+import javax.jcr.Property;
+import javax.jcr.PropertyIterator;
+import javax.jcr.RepositoryException;
+import javax.jcr.Value;
+
+public class ArcJcrContentNodeBinder extends DefaultJcrContentNodeBinder {
+
+    private static final Logger log = LoggerFactory.getLogger(ArcJcrContentNodeBinder.class);
+    private List<ExternalStorageReference> references;
+
+    @Override
+    protected Value[] createJcrValuesFromContentProperty(Node jcrNode, ContentProperty contentProp, ContentValueConverter<Value> valueConverter) throws RepositoryException {
+        Value[] vals = super.createJcrValuesFromContentProperty(jcrNode, contentProp, valueConverter);
```

Comment:
I think you should be able to just return this line without having to create the vals variable